src/coreclr/inc/winwrap.h

```diff
@@ -260,7 +255,7 @@ inline DWORD GetMaxDBCSCharByteSize()
 }
 
 #ifndef HOST_UNIX
-BOOL RunningInteractive();
+#define RunningInteractive() TRUE
```

Comment:
Could we inline `#ifdef HOST_WIN32` at the only remaining usage of RunningInteractive, and delete this inverted condition? https://github.com/dotnet/runtime/blob/ec349cd8e9960ec44bf528eef52c56a87e601207/src/coreclr/utilcode/utilmessagebox.cpp#L201

Reply:
Good idea. I will do that.